v/vanilla_bean/parameters.vh

```diff
@@ -235,4 +235,18 @@ localparam RV32_Jimm_width_gp     = 20;
 
 
 
+/////////////////////////////////////////////////////
+// The CUDA Print stat parameters
+// The cuda print stat tag message incorporates within itself the x,y coordiantes
+// and the tile group id of the tile that triggerd the bsg_cuda_print_stat instruction
+// <stat_type>   -   <y cord>   -   <x cord>   -   <tile group id>   -   <tag>
+localparam RV32_cuda_print_stat_tag_width_gp    = 8;
```

Comment:
parameters.vh is only for hardware parameters related to vanilla core, I wouldn't put this here.
Also, instead of having these lists of parameters, how about just defining a fixed-width struct?

Reply:
That's a good idea, I'm not sure how to do it correctly though. Let's discuss this tomorrow, thanks!

Reply:
Fixed, thanks to Tommy!